typescript/environments.d.ts

```diff
@@ -56,13 +56,24 @@ export type FormsAppEnvironmentSendingAddressResponse =
   }
 
 export interface BaseFormsAppEnvironment {
+  /** Name of the Forms App Environment */
   name: string
   description?: string
+  /** The exact organisation identifier the forms app environment is associated with */
   organisationId: string
+  /** Unique domain safe text to identify the forms app environment */
   slug: string
+  /**
+   * Array of emails addresses to be notified when an error occurs in processing
+   * submission events
+   */
   notificationEmailAddresses: string[]
   styles?: FormsAppEnvironmentStyles
   sendingAddress?: FormsAppEnvironmentSendingAddress
```

Comment:
Why did some of these not get comments?

Reply:
They didn't already have comments elsewhere

Reply:
That is a satisfactory answer, I will hit approve and merge